test/component_test.go

```
@@ -0,0 +1,248 @@
+	}()
+
+
+	// // Wait for the DnsEndpoint to be updated with the LoadBalancer metadata
+	factory := dynamicinformer.NewDynamicSharedInformerFactory(dynamicClient, 0)
+	// Create an informer for the DNSEndpoint resource
+	informer := factory.ForResource(dnsEndpointGVR).Informer()
+
+
+	stopChannel := make(chan struct{})
+
+	informer.AddEventHandler(cache.ResourceEventHandlerFuncs{
```

golangci-lint (1.64.8)

Error return value of `informer.AddEventHandler` is not checked

(errcheck)